common/web/recorder/src/index.ts

```diff
@@ -568,7 +570,7 @@ export class EventSpecTestSet implements TestSet<InputEventSpecSequence> {
     let testSet = this.testSet;
 
     for(var i=0; i < testSet.length; i++) {
-      var testSeq = this[i];
+      var testSeq = this.testSet[i];
```

Comment:
Is this a bug fix?

Reply:
Yes.  Once I saw it, I started wondering how (or even, if) it worked before.